plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -159,6 +161,26 @@ const fetchPluginData = async ({createNode, reporter}) => {
         page = pluginsContainer.page + 1;
     } while (!page || pluginsContainer.page < pluginsContainer.pages);
     await Promise.all(promises);
+    const updateUrl = 'https://updates.jenkins.io/current/update-center.actual.json';
+    const updateData = await requestGET({url: updateUrl, reporter});
+    const suspendedPromises = [];
+    for (const deprecation of Object.keys(updateData.deprecations)) {
+        if (names.indexOf(deprecation) == -1) {
+            suspendedPromises.push(createNode({
+                wiki: updateData.deprecations[deprecation],
+                id: deprecation,
+                name: deprecation,
+                suspended: true,
+                parent: null,
+                children: [],
+                internal: {
+                    type: 'JenkinsPlugin',
```

Comment:
Lets make this a completely different type. It only needs name, id and url right? then you can just loop through the new type in gatsby-node.js

Just simplifies things and keeps them seperate